src/main/java/uk/gov/hmcts/reform/civil/service/PaymentRequestUpdateCallbackService.java

```diff
@@ -46,44 +52,59 @@ public void processCallback(ServiceRequestUpdateDto serviceRequestUpdateDto) {
             CaseDetails caseDetails = coreCaseDataService.getCase(Long.valueOf(serviceRequestUpdateDto
                                                                                    .getCcdCaseNumber()));
             CaseData caseData = caseDetailsConverter.toCaseData(caseDetails);
-            caseData = updateCaseDataWithStateAndPaymentDetails(serviceRequestUpdateDto, caseData);
-            createEvent(caseData, SERVICE_REQUEST_RECEIVED,
-                        serviceRequestUpdateDto.getCcdCaseNumber()
-            );
+            caseData = updateCaseDataWithStateAndPaymentDetails(serviceRequestUpdateDto, caseData, feeType);
+            if (feeType.equals(FeeType.HEARING.name()) || feeType.equals(FeeType.CLAIMISSUED.name())) {
+                createEvent(caseData, serviceRequestUpdateDto.getCcdCaseNumber(), feeType);
+            }
 
         } else {
-
-            log.error("Case id {} not present", serviceRequestUpdateDto.getCcdCaseNumber());
+            log.error("Service request status is not PAID for Case id {}",
```

Comment:
`log.error` is intended for application errors, not to report that a payment didn't go through.

Either remove this else or log the information at `info` level.

Reply:
Moved to info level - resolved myself, as you two are still off on sick leave.